Source/Plugins/Core/com.equella.base/src/com/tle/common/search/DefaultSearch.java

```diff
@@ -404,29 +404,26 @@ private boolean anyEmpty(Collection<?>... collections) {
   }
 
   public static class QueryParser {
-    private final List<QueryToken> tokens;
+
     private final List<String> hilight;
```

Comment:
is this name specific to a library?
If not spelling it out fully could be nice

```suggestion
    private final List<String> highlight;
```

Reply:
Heh! :grin: Yeah I also was sooooo tempted to go and change this, but then I'd also need to modify the `getHilightedList()` and then all the other calls to it. Suddenly it seemed like I'd be getting too carried away for this PR. I wonder if it's a typo or a historical something or rather. :thinking: